envrc.el

```diff
@@ -363,3 +364,5 @@ This can be useful if a .envrc has been deleted."
 
 (provide 'envrc)
 ;;; envrc.el ends here
+
+; LocalWords:  envrc direnv
```

Comment:
What's this one about? Haven't seen it before, and it's not in a `Local Variables` block.

Reply:
This indicates words that ispell should just accept without complaining. Check the docstring of `ispell-words-keyword`.

Reply:
You might also want to check `checkdoc-spellcheck-documentation-flag`.